### PR TITLE
Expand Info about What People can do with DOT

### LINKED ISSUE #4119: Expand Section about What PPL can do with DOT
- [x] 1 dot to join nomination pools
- [x] 200 dots to create a pool
- [x] 20 dots create proxy account

### PR DESCRIPTION
**Pull request by filippoweb3**
See linked issue

### CODE REVIEW

**Comment by DrW3RK** [comment]
@alfarok, Can we automatically fetch the minimum active nomination from the chain and display it here? IMO, it adds a lot of value to this page on the Wiki to ensure that the reader is not misled into believing 10 DOT is enough to become an active nominator and earn rewards.

**Comment by DrW3RK** [comment]
You can either implement it in this PR or merge this PR for now and work on it in a standalone PR.

File: docs/general/getting-started.md [hunk 1]
```diff
@@ -176,17 +176,25 @@ functionalities on Polkadot.
...
-- **10 DOT**: the minimum stake required to submit your intent to nominate validators. But, to earn
-  staking rewards, your stake should not be less than the minimum stake among the active nominators
-  which is a dynamic threshold. Learn more about
-  [becoming a nominator](../learn/learn-nominator.md).
-
-- **20 DOT**: you can register an [on-chain identity](../learn/learn-identity.md) or vote for
-  [council](../maintain/maintain-guides-how-to-vote-councillor.md).
+- **10 DOT**: the minimum stake required to submit your intent to directly nominate validators. But, to
+  increase the chance of earning staking rewards, your stake should not be less than the minimum
+  stake among the active nominators which is a dynamic threshold (which is around 210 DOT in era 911).
```
Comment (alfarok) [inline docs/general/getting-started.md]:
```suggestion 
stake among the active nominators which is a dynamic threshold (around 210 DOT in era 911).
```

**Review by alfarok — APPROVED** [review]
LGTM :+1: , committing one suggestion that removes duplicate phrasing ("which is" used twice) and merging

**Comment by alfarok** [comment]
> @alfarok, Can we automatically fetch the minimum active nomination from the chain and display it here? IMO, it adds a lot of value to this page on the Wiki to ensure that the reader is not misled into believing 10 DOT is enough to become an active nominator and earn rewards.

I am going to break this out into a new follow-up issue, #4123.

**Pull request merged**